internal-api/src/main/kotlin/ua/anastasiia/finesapp/NatsSubject.kt

```diff
@@ -17,33 +13,21 @@ object NatsSubject {
 
     object TrafficTicket : Subject("trafficticket") {
         private const val ADDED = ".added"
-
         val ADD = inputReqReply("add")
-
-        fun getAddedEventSubject(carPlate: String): String {
-            return "${outputPubSub(carPlate)}$ADDED"
-        }
+        fun eventSubject(carPlate: String) = outputPubSub(carPlate, ADDED)
```

Comment:
```suggestion
        fun addedSubject(carPlate: String) = outputPubSub(carPlate, ADDED)
```
since later events about updated, deleted may be added